usr/share/handygccs/scripts/handycon.py

```diff
@@ -634,10 +637,13 @@ async def capture_keyboard_events():
                             # This device class uses the same active events with different values for AYA SPACE, LC, and RC.
                             if active == [97, 125]:
 
-                                # LC | Default: Screenshot
+                                # LC | Default: Screenshot / Launch Chimera
                                 if button_on == 102 and event_queue == []:
-                                    event_queue.append(button1)
-                                    this_button = button1
+                                    if HAS_CHIMERA_LAUNCHER:
+                                        launch_chimera()
```

Comment:
You'll need to event_queue.append() an event here. Adding one for "button7" in the above map that is just a string array like ["launch chimera"]  should be fine. Then check for it in the below release event to event_queue.remove(). This will prevent multiple activation's of this event.